docker-compose/docker-compose-esignet/config/esignet-default.properties

```diff
@@ -523,7 +523,7 @@ mosip.esignet.vci.key-values={\
                       'format': 'ldp_vc',\
                       'scope' : 'life_insurance_vc_ldp',\
                       'cryptographic_binding_methods_supported': {'did:jwk'},\
-                      'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                      'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
                       'proof_types_supported': {'jwt'},\
                       'credential_definition': {\
                           'type': {'VerifiableCredential'},\
```

Comment:
As per the version 12, there is support for authorization_servers. 
We should authorization_servers in our metadata config files that since certify and eSignet are 2 different systems.